boards/seeed/xiao_nrf54l15/xiao_nrf54l15_nrf54l15_cpuapp.dts

```diff
@@ -0,0 +1,311 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf54l15_cpuapp.dtsi>
+#include "xiao_nrf54l15-pinctrl.dtsi"
+#include <zephyr/dt-bindings/sensor/lsm6dso.h>
+
+/ {
+	model = "Custom Board auto generated by nRF Connect for VS Code";
+	compatible = "seeed,xiao-nrf54l15-cpuapp";
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
```

Comment:
likewise newline caps between compatible and child nodes, line 16, fix in whole PR